src/brain/apis/index.ts

```diff
@@ -13,6 +13,45 @@ type SlackBlock = {
     text: string;
   };
 };
+
+export function getMessageBlocks(response, team = ''): Array<SlackBlock> {
```

Comment:
Not new code. Just moving line 61+ to its own module so I can use it in SlackScores